apps/condo/domains/ticket/hooks/useTableColumns.tsx

```diff
@@ -196,7 +197,7 @@ export const useTableColumns = (sort: Array<string>, filters: IFilters,
                 render: (record) => {
                     const unitName = get(record, 'unitName')
                     const property = get(record, 'property')
-                    const text = get(property, 'address')
+                    const text = get(property, 'address', NoPropertyMessage) 
```

Comment:
When address is a blank string `''`, `NoPropertyMessage` is not assigned and the table cell is blank.
<img width="314" alt="image" src="https://user-images.githubusercontent.com/2064830/125297801-3e69f980-e330-11eb-8460-7e1c59556531.png">